### DISCUSSION TITLE
Changing the value of the CreatedUtc field for a ContentItem object

### DISCUSSION THREAD

**Discussion by juniordev19** [OrchardCMS/OrchardCore#15846] [Q&A]
Is it possible to change the CreatedUTC value of ContentItem?
I'm trying to change this field in the database table, but the new value is not synchronized with the site.

**Comment by Skrypt** [comment]
You need to use the CommonPart on the Content Type and modify the CreatedDate from that CommonPart since it is how we permit changing the Owner and Creation Date of a content item.

**Comment by juniordev19** [comment]
How should I do this if I installed the Orchard Core as a nuget package?

**Comment by Skrypt** [comment]
You need to change the CreatedUtc value in the ContentItemIndex table (where the data is mapped) and also in the Document Content Column (where the data is reduced as JSON).

**Comment by sebastienros** [comment]
In the admin, open the Content Type Definitions, seek the one you want to change, then click "Add Parts" and then select "Common Part". Now when you edit one of these content items you should see a new editor that lets you change the Creation Date.
